inc/dbmysqliterator.class.php

```diff
@@ -379,22 +377,30 @@ function buildQuery ($table, $crit = "", $log = false) {
     * @reutn string
     */
    public function handleOrderClause($clause) {
-      if (!is_array($clause)) {
-         $clause = [$clause];
-      }
-
       $cleanorderby = [];
-      foreach ($clause as $o) {
-         $fields = explode(',', $o);
-         foreach ($fields as $field) {
-            $new = '';
-            $tmp = explode(' ', trim($field));
-            $new .= DBmysql::quoteName($tmp[0]);
-            // ASC OR DESC added
-            if (isset($tmp[1]) && in_array($tmp[1], ['ASC', 'DESC'])) {
-               $new .= ' '.$tmp[1];
+      if ($clause instanceof \QueryExpression) {
```

Comment:
You should add `QueryExpression` as possible type for `$clause` param in doc block.